### ISSUE TITLE
Changing the mod key

### ISSUE THREAD

**Issue by KriiV** [ianyh/Amethyst#415]
Just wondering if it is possible to change the mod key to something different?

Thanks!

[Trello Card](https://trello.com/c/g976muq2/204-amethyst-changing-the-mod-key)

**Comment by nanzhong** [comment]
I'm also wondering about this. I've tried setting this via `.amethyst`:
```
{
    "mod1": [
        "command",
        "shift"
    ],
    "mod2": [
        "command",
        "shift",
        "control"
    ]
}
```

but that dosen't seem to work either.

**Comment by nanzhong** [comment]
Just a heads up I'm using 0.10b2 that up upgraded from 0.9.10 installed via homebrew.

After a little bit of digging it seems like there's something weird going on in the loading of config + NSUserDefaults stored settings.

I was able to modify the mod keys by:
1. Quit amethyst
2. Delete the NSUserDefaults settings: `rm ~/Library/Preferences/com.amethyst.Amethyst.plist`
3. Modify the default config inside the amethyst bundle (for me this was located at `/opt/homebrew-cask/Caskroom/amethyst/0.9.10/Amethyst.app/Contents/Resources/default.amethyst`) to whatever I wanted
3. Restart amethyst

**Comment by ianyh** [comment]
That should definitely work. I likely introduced something in the 0.10 beta.

**Comment by ianyh** [comment]
I think this may fixed by the fix for #389.
